packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -174,20 +174,27 @@ class OnsetPicker extends React.Component {
    * @param {event} - Triggered change event
    */
   changeAge(event) {
-    const age = Number(event.target.value);
-
-    this.setState((prevState) => {
+    if (event.target.value !== '') {
+      const age = Number(event.target.value);
```

Comment:
we can declare this once at the beginning use LET and can be reused in else block also.

Reply:
in else block you can remove age : [here](https://github.com/cerner/terra-clinical/blob/5d0df5ef328f6245d74e7c707014a9ebb166f02e/packages/terra-clinical-onset-picker/src/OnsetPicker.jsx#L192) and can set undefined [here](https://github.com/cerner/terra-clinical/blob/5d0df5ef328f6245d74e7c707014a9ebb166f02e/packages/terra-clinical-onset-picker/src/OnsetPicker.jsx#L194)

Reply:
Resolved in 7afc761